src/utils/misc.cpp

```diff
@@ -155,9 +155,11 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath) {
                 QStringLiteral("nautilus"),
                 QStringList{"--no-desktop", QDir::toNativeSeparators(path)});
         } else if (output == QStringLiteral("caja-folder-handler.desktop")) {
+            QDir dir = QFileInfo(path).absoluteDir();
+            QString absDir=dir.absolutePath();
```

Comment:
Thanks for the patch.
Can you make these both `const`?
and fix the formatting on second line?

Reply:
Like this?